cmd/edgectl/main.go

```diff
@@ -15,7 +16,9 @@ import (
 var Version = "(unknown version)"
 
 const socketName = "/var/run/edgectl.socket"
-const logfile = "/tmp/edgectl.log"
+
+var logfile = filepath.Join(os.TempDir(), "edgectl.log")
```

Comment:
What does this yield on a Mac? This needs to be a user-friendly path.

Reply:
BTW, this is _also_ a machine-scoped file. Is this change really appropriate?

Reply:
I guess not.  Removing that bit from the commit.